platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,112 @@
+
+**Xpra 6.2.0**
+
+[Xpra](https://github.com/Xpra-org/xpra), known as _screen for X_, allows you to run X11 programs by giving you remote access to individual graphical applications. The container template image also installs NVIDIA Linux x64 (AMD64/EM64T) drivers for Ubuntu 22.04 for running GPU-enabled applications. To use these GPU drivers, make sure that your compute environment specifies GPU instance families.
+
+The default user is the `root` account. The image is based on `ubuntu:jammy`. Additional system-level packages can be installed in a terminal window using `apt install <package_name>`.
+
+## Session statuses
+
+Data studios have the following possible statuses:
+
+- **building**: When a custom environment is building the template image for a new data studio session. The [Wave] service performs the build action. For more information on this status, see [Inspect custom container template build status][build-status].
+- **build-failed**:  When a custom environment build has failed. This is a non-recoverable error. Logs are provided to assist with troubleshooting. For more information on this status, see [Inspect custom container template build status][build-status].
+- **starting**: The data studio is initializing.
+- **running**: When a data studio session is **running**, you can connect to it, copy the data studio URL, or stop it. In addition, the session can continue to process requests/run computations in the absence of an ongoing connection.
+- **stopping**: The recently-running session is in the process of being stopped.
+- **stopped**: When a session is stopped, the associated compute resources are deallocated. You can start or delete the data studio when it's in this state.
+- **errored**: This state most often indicates that there has been an error starting the data studio session but it is in a **stopped** state. There might be errors reported by the session itself but these currently will be overwritten with a **running** status if the data studio session is still running.
+
+  :::tip
+  If you encounter an error with the public preview release of Data Studios [contact Seqera support][contact].
+  :::
+
+## Session checkpoints
+
+When you start a session, it automatically creates a *checkpoint*. A checkpoint saves changes that you make to the root filesystem and stores it in the compute environment's pipeline work directory in the `.studios/checkpoints` directory under a unique name. The checkpoint is updated every five minutes.
+
+When you stop and start a data studio session, or start a new data studio session from a previously created checkpoint, changes such as installed software packages and configuration files are restored and made available in the data studio session. Changes made to mounted datasets are not included in a checkpoint.
```

Comment:
Mounted datasets, or just mounted data?

Reply:
@robnewman, do we make the distinction? Thanks!

Reply:
Let's change it to "mounted data", @jason-seqera.